health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/MapDrillChart.js

```
@@ -0,0 +1,135 @@
+import { format } from "date-fns";
+
+
+const Backsvg = ({ onClick }) => (
+    <svg
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+      fill="none"
+      onClick={onClick}
+      xmlns="http://www.w3.org/2000/svg"
+    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)